pkg/artifact_builder/docker_compose.go

```diff
@@ -47,7 +51,15 @@ func (bc *BuilderConfig) invokeDockerCompose(command DockerCommand) ([]byte, err
 
 	envVars := bc.GetBuildEnv()
 	envVars = append(envVars, os.Environ()...)
-	envVars = append(envVars, "DOCKER_BUILDKIT=1")
```

Comment:
why remove this one? seems like we actually want it and might resolve your error you mentioned in the other comment regarding buildkit?